pkg/storaged/client.js

```diff
@@ -505,7 +505,15 @@ function init_model(callback) {
     Promise.allSettled([client.manager.wait(),
         client.mdraids.wait(), client.vgroups.wait(), client.drives.wait(),
         client.blocks.wait(), client.blocks_ptable.wait(), client.blocks_lvm2.wait(), client.blocks_fsys.wait()
-    ]).then(() => {
+    ]).then(results => {
+        // we at least need the manager object; if it doesn't exist, wait for the next proxy onchanged
```

Comment:
A "onchanged" event doesn't rerun feature detection, no?  The only way to get out of "features = false" is to reload the page.

Reply:
It does. It calls `init_client()`.

Reply:
Right. That's in fact what has hidden the B-Bus race in many cases.

Reply:
> It does. It calls `init_client()`.

Oh, your are right. That changes everything, sorry for missing it.